mail_debrand/models/mail_mail.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 O4SB - Graeme Gellatly
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# Copyright 2020 Onestein - Andrea Stirpe
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class MailMail(models.AbstractModel):
+    _inherit = "mail.mail"
+
+    # in messages from objects is adding using Odoo that we are going to remove
+
+    @api.model_create_multi
+    def create(self, values_list):
+        for index, _value in enumerate(values_list):
+            values_list[index]["body_html"] = self.env[
```

Comment:
You don't need to enumerate:
```suggestion
        for vals in values_list:
            vals["body_html"] = self.env[
```